internal/pkg/template/templates/cicd/buildspec.yml

```
@@ -108,6 +108,12 @@ phases:
             if [ -n "$build_context" ]; then
               df_dir_path=$build_context
             fi
+            platform_string=$(echo $manifest | jq -r '.platform // ""')
+            platform_os=$(echo $manifest | jq -r '.platform?.osfamily? // ""')
+            platform_arch=$(echo $manifest | jq -r '.platform?.architecture? // ""')
+            if [ -n "$platform_os" -a "$platform_arch" ]; then
```

Is it equivalent to 
```suggestion
            if [ "$platform_os" -a "$platform_arch" ]; then
```
https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html